bigquery_etl/cli/query.py

```diff
@@ -1177,34 +1179,64 @@ def initialize(name, sql_dir, project_id, dry_run):
         sys.exit(1)
 
     for query_file in query_files:
-        init_files = Path(query_file.parent).rglob("init.sql")
-        client = bigquery.Client()
-
-        for init_file in init_files:
-            project = init_file.parent.parent.parent.name
-
-            with open(init_file) as init_file_stream:
-                init_sql = init_file_stream.read()
-                dataset = Path(init_file).parent.parent.name
-                job_config = bigquery.QueryJobConfig(
-                    dry_run=dry_run,
-                    default_dataset=f"{project}.{dataset}",
-                )
-
-                if "CREATE MATERIALIZED VIEW" in init_sql:
-                    click.echo(f"Create materialized view for {init_file}")
-                    # existing materialized view have to be deleted before re-creation
-                    view_name = query_file.parent.name
-                    client.delete_table(
-                        f"{project}.{dataset}.{view_name}", not_found_ok=True
+        sql_content = query_file.read_text()
```

Comment:
@scholtzan this adds the capability to initialize a query file. It first deploys the schema, then runs the query with the templated `is_init` set to `True`.

This feels a little hacked together, and is probably not well tested.

Reply:
Ah missed this. Yes some tests would be good. I'll file a ticket.